documentation/API-design-guidelines.md

```diff
@@ -1342,17 +1345,20 @@ Following Error code must be present:
 
 ##### Termination for resource-based (explicit) subscription
 
-3 scenarios of subscription termination are possible (business conditions may apply):
+4 scenarios of subscription termination are possible (business conditions may apply):
 
 * case1: subscriptionExpireTime has been provided in the request and reached. The operator in this case has to terminate the subscription.
-* case2: subscriber requests the `DELETE` operation for the subscription (if the subscription did not have a subscriptionExpireTime or before subscriptionExpireTime expires). 
-* case3: subscription ends on operator request. 
+* case2: subscriptionMaxEvents has been provided in the request and reached. The operator in this case has to terminate the subscription.
+* case3: subscriber requests the `DELETE` operation for the subscription (if the subscription did not have a subscriptionExpireTime or before subscriptionExpireTime expires). 
+* case4: subscription ends on operator request. 
 
-It could be useful to provide a mechanism to inform subscriber for case3 (and probably case1). In this case, a specific event type could be used.
+It could be useful to provide a mechanism to inform subscriber for all case. In this case, a specific event type could be used.
 
-_Termination rules regarding subscriptionExpireTime usage_
-* When client side providing a `subscriptionExpireTime`, service side has to terminate the subscription without expecting a `DELETE` operation.
-* When the `subscriptionExpireTime` is not provided, client side has to trigger a `DELETE` operation to terminate it.
+_Termination rules regarding subscriptionExpireTime & subscriptionMaxEvents usage_
+* When client side providing a `subscriptionExpireTime` and/or `subscriptionMaxEvents` service side has to terminate the subscription without expecting a `DELETE` operation.
+* If both `subscriptionExpireTime` and `subscriptionMaxEvents` are provided, the subscription will end when the first one is reached.
+* When the `subscriptionExpireTime` and `subscriptionMaxEvents` are not provided, client side has to trigger a `DELETE` operation to terminate it.
```

Comment:
```suggestion
* When the `subscriptionExpireTime` and/or `subscriptionMaxEvents` are not provided, client side has to trigger a `DELETE` operation to terminate it.
```

Reply:
Hi @PedroDiez
Thanks a lot for your review always helpful.

I'm not sure of this comments :)  For me only if the `subscriptionExpireTime` **and** `subscriptionMaxEvents` are not provided that consumer side has to use the DELETE operation to terminate the subscription. WDYT ?

Reply:
Hi @bigludo7, yes you are right (it is clear and condition). Let me suggest some refinement

Reply:
Hi Ludovic, I have committed this comment.
LGTM